framework/src/modules/network/network.js

```diff
@@ -69,63 +84,83 @@ module.exports = class Network {
 				}`
 			);
 		});
+
 		this.p2p.on(EVENT_CONNECT_OUTBOUND, peerInfo => {
 			this.logger.info(
 				`Connected to peer ${peerInfo.ipAddress}:${peerInfo.wsPort}`
 			);
 		});
+
 		this.p2p.on(EVENT_DISCOVERED_PEER, peerInfo => {
 			this.logger.info(
 				`Discovered peer ${peerInfo.ipAddress}:${peerInfo.wsPort}`
 			);
 		});
+
 		this.p2p.on(EVENT_FAILED_TO_FETCH_PEER_INFO, error => {
```

Comment:
I was about to fix it in my PR as it was debugging a lot of undefined messages. 👍